package.json

```diff
@@ -24,8 +24,10 @@
     "yarn": ">= 1.0.0"
   },
   "scripts": {
-    "start": "node server/app.js",
-    "test": "grunt test",
+    "start": "webpack-serve",
+    "build": "webpack --config webpack.config.js",
```

Comment:
By default `webpack` is looking to a configuration file `webpack.config.js` so why not using instead:

```js
…
"build": "webpack",
…
```

Reply:
done ;)